README.md

```diff
@@ -68,13 +68,13 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](-----------------------)
+- [Live Demo Link](https://rushanika.onrender.com/)
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 <!-- VIDEO  PRESENTATION -->
 
 ## 🚀 Video presentation <a name="video-demo"></a>
 
-- [Video Link](-------------)
+- [Video Link](https://www.loom.com/share/0631c1e63511414d90646f66bcc62821)
```

Comment:
- [ ] Also, as requested by the previous reviewer, please make sure to include some content in the `Future features` section of your README file. Currently, it is empty:

![image](https://github.com/josuerushanika/budget-app/assets/77069011/e87bcfee-e45a-417a-b406-4db781835299)

Consider implementing this! 🤓 🔧